src/Bicep.LangServer.UnitTests/Handlers/BicepEditLinterRuleCommandHandlerTests.cs

```diff
@@ -163,11 +164,46 @@ public async Task IfConfigExists_AndContainsRuleAlready_ThenJustShowAndSelect()
             ev.Properties.Should().Contain(new Dictionary<string, string> {
                     { "code", "no-unused-params" },
                     { "newConfigFile", "false" },
+                    { "newRuleAdded", "false" },
                     { "error", string.Empty },
                     { "result", Result.Succeeded },
                 });
         }
 
+        [TestMethod]
+        public async Task IfConfigExists_AndContainsRuleButNotLevel_ThenJustAddLevelAndSelect()
```

Comment:
Can we verify telemetry properties for this test as well?

Reply:
No reason to do it in every test.  It's already effectively tested.